.gitignore

```diff
@@ -17,31 +17,18 @@ share/python-wheels/
 
 # Jupyter Notebook
 .ipynb_checkpoints
-nipoppy/notebooks/.ipynb_checkpoints/*
 
-# dataset-specific files
-bids_filter.json
-workflow/proc_pipe/fmriprep/scripts/fmriprep_sge.sh
-workflow/proc_pipe/fmriprep/scripts/fmriprep_slurm.sh
-workflow/dicom_org/dicom_dir_func.py
-
-# logs/testing
-*.out
-*.log
+# tests
 .coverage
 htmlcov
 
-
-env/
```

Comment:
I would opt to leave that one in. Maybe adding `.venv` and `venv` as well. Just to prevent adding virtual env by mistake